docs/howto/customize/configure-login-page.md

```diff
@@ -6,4 +6,8 @@ For an example, see [the log-in page of the staging hub](https://staging.pilot.2
 
 The log-in pages are built with [the base template at this repository](https://github.com/2i2c-org/pilot-homepage). Values are inserted into each template based on each hub configuration.
 
-You may customize the configuration for a hub's homepage at `config.jupyterhub.homepage.templateVars`. Changing these values for a hub will make that hub's landing page update automatically. We recommend [using the `hubs.yaml` file as a reference](https://github.com/2i2c-org/pilot-hubs/blob/master/hubs.yaml). Copy the configuration from another hub, and then modify the contents to fit the new hub that is being configured.
+You may customize the configuration for a hub's homepage at
+`config.jupyterhub.homepage.templateVars`. Changing these values for a hub
+will make that hub's landing page update automatically. 
+Copy the configuration from another hub, and then modify the contents to fit
+the new hub that is being configured.
```

Comment:
```suggestion
You may customize the configuration for a hub's homepage at `config.jupyterhub.homepage.templateVars`. Changing these values for a hub will make that hub's landing page update automatically.
Copy the configuration from another hub, and then modify the contents to fit the new hub that is being configured.
```

Not blocking on this PR, but what do folks think about adopting a "one line per sentence" documentation policy? I find that this strikes a nice balance between "gigantic lines because a whole paragraph is in it" and "shorter lines but with broken up sentences"

Reply:
ah, hmm - my text editor can automatically either reflow lines to match a max width, or automatically wrap them so they don't horizontally scroll. However, with this reflow will have to be manual - how do you deal with that in your writing workflow?

I'm also curious how to make this determination without it becoming a timesuck bikeshed :D

Reply:
haha yeah - for me it has just become natural when I'm writing documentation - I finish a sentence then hit enter. I basically follow the first few steps of https://sembr.org/ but don't spend the time worrying about the whole "split concepts into separate lines" parts of it.

Reply:
Other than the "make the source code easier to semantically follow" part, I also like the fact that it increases the likelihood that I can suggest an edit with a one-liner in the github PR interface (because edits often happen within-sentence, and the whole sentence would be on the same line)

Reply:
either way, please do not block this PR on these thoughts!